packages/gafl-webapp-service/src/handlers/__tests__/page-handler.spec.js

```diff
@@ -206,6 +206,15 @@ describe('The page handler function', () => {
       })
     )
   })
+
+  it('the post method calls redirectWithLanguageCode correctly when completion is not a function', async () => {
```

Comment:
I think a better name would be something like 'post method called redirectWithLanguageCode with completion value when completion is a string'. Might also be a candidate for it.each, using different completion urls (then you could replace 'completion value' with the url being used.)